From 790641f6b02892485fc295ff468ed8ca0e8b96cf Mon Sep 17 00:00:00 2001
From: Sylvain Corlay <sylvain.corlay@gmail.com>
Date: Wed, 28 Nov 2018 16:52:25 +0100
Subject: [PATCH] Imported libraries visibility

---
 cmake/FindIlmBase.cmake | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/cmake/FindIlmBase.cmake b/cmake/FindIlmBase.cmake
index 3501c1f..67c38f5 100644
--- a/cmake/FindIlmBase.cmake
+++ b/cmake/FindIlmBase.cmake
@@ -168,7 +168,7 @@ foreach(ILMBASE_LIB
     IlmThread)
 
     if (OPENEXR_${ILMBASE_LIB}_LIBRARY)
-        add_library(IlmBase::${ILMBASE_LIB} SHARED IMPORTED)
+        add_library(IlmBase::${ILMBASE_LIB} STATIC IMPORTED GLOBAL)
         set_target_properties(IlmBase::${ILMBASE_LIB} PROPERTIES IMPORTED_LOCATION_RELEASE ${OPENEXR_${ILMBASE_LIB}_LIBRARY})
         set_target_properties(IlmBase::${ILMBASE_LIB} PROPERTIES IMPORTED_LOCATION_DEBUG ${OPENEXR_${ILMBASE_LIB}_DEBUG_LIBRARY})
         set_target_properties(IlmBase::${ILMBASE_LIB} PROPERTIES MAP_IMPORTED_CONFIG_RELWITHDEBINFO RELEASE)
@@ -177,7 +177,7 @@ foreach(ILMBASE_LIB
     endif()
 
     if (OPENEXR_${ILMBASE_LIB}_STATIC_LIBRARY)
-        add_library(IlmBase::${ILMBASE_LIB}_static SHARED IMPORTED)
+        add_library(IlmBase::${ILMBASE_LIB}_static STATIC IMPORTED GLOBAL)
         set_target_properties(IlmBase::${ILMBASE_LIB}_static PROPERTIES IMPORTED_LOCATION_RELEASE ${OPENEXR_${ILMBASE_LIB}_STATIC_LIBRARY})
         set_target_properties(IlmBase::${ILMBASE_LIB}_static PROPERTIES IMPORTED_LOCATION_DEBUG ${OPENEXR_${ILMBASE_LIB}_STATIC_DEBUG_LIBRARY})
         set_target_properties(IlmBase::${ILMBASE_LIB}_static PROPERTIES MAP_IMPORTED_CONFIG_RELWITHDEBINFO RELEASE)
-- 
2.18.0

